digdag-standards/src/main/java/io/digdag/standards/command/kubernetes/KubernetesClientConfig.java

```diff
@@ -19,40 +19,33 @@ public static KubernetesClientConfig create(final Optional<String> name,
             final Config systemConfig,
             final Config requestConfig)
     {
-        if (requestConfig.has("kubernetes")) {
+        Config config;
+        if (requestConfig.get("agent.command_executor.type", String.class, "").equals("kubernetes")) {
```

Comment:
Could you explain why you change config key name from "kubernetes" to "agent..." ?
I guess original code is better. Because Docker has same configuration naming. (Check "docker")
And "agent..." is longer to define in WF definition.

Reply:
@yoyama 
Review Thank you.
I misunderstood the mechanism a little.
Changed to use "Kubernetes".